Dangerfile

```diff
@@ -1,14 +1,16 @@
 # Reference: http://danger.systems/reference.html
```

Comment:
That's an awesome tool, we have to think about adding it to our projects!

Reply:
Good point, definitely worth adding it elsewhere 👍